### PR TITLE
Split 1q and 2q tables into different dataframes

### PR DESCRIPTION
**Pull request by GuillermoAbadLopez**
This PR splits the 1q and 2q tables, and since lot of code was going to be duplicated, I have refactored lot of code in methods, to be called twice, making also the code more readable 🥇 

The final result tables look like this:
Screenshot 2024-01-16 at 18 33 43

### CODE REVIEW

File: src/qililab/calibration/calibration_controller.py [hunk 1]
```diff
@@ -642,80 +652,127 @@ def get_last_fidelities(self) -> pd.DataFrame:
...
                 for bus, _, parameter, value in node.output_parameters["platform_parameters"]:
                     bus_list = str(bus).split("_")
                     bus = "_".join([x for x in bus_list if not any(char.isdigit() for char in x)])
                     df[f"{str(parameter)}_{bus}"][qubit] = value
 
-        # Reorder fidelities to the front of the dataframe:
+        return df
+
+    @staticmethod
+    def _reorder_fidelities(df: pd.DataFrame) -> pd.DataFrame:
```
Comment (jjmartinezQT) [inline src/qililab/calibration/calibration_controller.py:767]: Is this for display purposes or we need them at front for the cloud team database format?

Comment (GuillermoAbadLopez) [reply src/qililab/calibration/calibration_controller.py:767]: Display purposes 👌

Comment (jjmartinezQT) [reply src/qililab/calibration/calibration_controller.py:767]: Cool, thanks!

**Review by jjmartinezQT — APPROVED** [review]
Amazing job, ran this one myself for full autocalibration without issues and getting tables in right format.

**Pull request merged**